pom.xml

```diff
@@ -411,11 +406,6 @@
                         <groupId>javax.media</groupId>
                         <artifactId>jai_imageio</artifactId>
                     </exclusion>
-                    <!-- this lib is also used in snap-cluster-analysis and in jlinda-core -->
```

Comment:
I would still exclude the ejml lib here explicitly.
Otherwise we might still get two versions on the class path
```
<!-- this lib is also used in snap-cluster-analysis and in jlinda-core
 better have an explicit dependency than rely on the transitive dependency
-->
<exclusion>
    <groupId>org.ejml</groupId>
    <artifactId>ejml-core</artifactId>
</exclusion
<exclusion>
    <groupId>org.ejml</groupId>
    <artifactId>ejml-ddense</artifactId>
</exclusion
```